_rules/SC3-1-2-lang-valid.md

```diff
@@ -19,15 +19,19 @@ authors:
 
 ### Applicability
 
-Any DOM element, within the `body` of a webpage with a [non-empty](#non-empty) `lang` or `xml:lang` attribute.
+This rules applies to each HTML or SVG element that is a [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) of the `body` element and has a `lang` or `xml:lang` attribute that is not empty ("").
```

Comment:
This is the wrong type of "within the `body`". This has to be a descendant in the composed tree, not in the light DOM tree.

Reply:
*Flat tree, yes.